libsolidity/ast/Types.cpp

```diff
@@ -889,6 +889,9 @@ tuple<bool, rational> RationalNumberType::isValidLiteral(Literal const& _literal
 		case Literal::SubDenomination::Wei:
 		case Literal::SubDenomination::Second:
 			break;
+		case Literal::SubDenomination::Gwei:
+			value *= bigint("1000000000");
```

Comment:
What do you think about adding digit separators for readability?
```suggestion
			value *= bigint(1000'000'000);
```

Reply:
`1'000'000'000`